src/screens/General/Dashboard.js

```diff
@@ -348,25 +348,25 @@ class Dashboard extends Component {
 			black
 		} = styles;
 
-		let buttonLinks = [
-			["https://shpeucf2018-2019.slack.com/", "slack"],
-			["https://www.facebook.com/shpeucfchapter/", "facebook"],
-			["https://www.shpeucf.com/", "globe"],
-			["https://www.instagram.com/shpeucf/?hl=en", "instagram"]
+		let socialMediaLinks = [
+			{ appLink: ["slack", "channel", "announcements"], appImage: "slack" },
+			{ appLink: ["facebook", "groups", "shpeucf"], appImage: "facebook" },
+			{ appLink: ["web", "open", "https://www.shpeucf.com/"], appImage: "globe" },
+			{ appLink: ["instagram", "open", "shpeucf"], appImage: "instagram" }
 		];
 
 		return (
 			<View style = { socialMediaContainer }>
 				<View style = { buttonRowContainer }>
-					{ buttonLinks.map((data, index) =>
+					{ socialMediaLinks.map((linksArray, index) =>
```

Comment:
Why is linksArray called an array? Array shouldn't be in the name regardless but it doesn't seem to be an array as I see below you are accessing linksArray.appImage and linksArray.applink. I think this should just be called link or socialMediaLink